assets/ic_download.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="26" height="25" viewBox="0 0 26 25" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M1 6V24H25V6" stroke="white" stroke-width="2"/>
+<path d="M13.6623 13.9624L17.9103 9.71441L19.3245 11.1286L12.6622 17.7909L6 11.1286L7.41421 9.71441L11.6623 13.9625V2.52881e-07L13.6623 0V13.9624Z" fill="white"/>
```

Comment:
- [ ] [OPTIONAL] I would recommend using the Microverse README template and their MIT License instead of the currently used one.